librz/bin/bin.c

```diff
@@ -673,6 +673,35 @@ RZ_API RZ_BORROW RzBinMap *rz_bin_object_get_map_at(RZ_NONNULL RzBinObject *o, u
 	return NULL;
 }
 
+/**
+ * \brief Find the last binary symbol at offset \p off.
+ *
+ * This function returns the last binary symbol that contains offset \p off,
+ * assuming that symbols are sorted by priority, and the last one is the most
+ * important one.
+ *
+ * \param o Reference to the \p RzBinObject instance
+ * \param off Address to search
+ * \param va When false, the offset \p off is considered a physical address; otherwise, a virtual address
+ * \return Pointer to a \p RzBinSymbol containing the address
+ */
+
+RZ_API RZ_BORROW RzBinSymbol *rz_bin_object_get_symbol_at(RZ_NONNULL RzBinObject *o, ut64 off, bool va){
+	rz_return_val_if_fail(o, NULL);
+
+	RzBinSymbol *sym;
+	RzListIter *iter;
+	ut64 from, to;
+
+	rz_list_foreach (o->symbols, iter, sym) {
+		from = va ? rz_bin_object_addr_with_base(o, sym->vaddr) : sym->paddr;
+		to = from + (va ? sym->size : sym->size);
```

Comment:
this seems wrong.

Reply:
In the issue, It was asked to implemente it like rz_bin_object_get_map_at() and rz_bin_object_get_string(). I somehow tried, but can you assist in resolving the issue

Reply:
if VA just compare it with the vaddr, if paddr, then just compare it with paddr.